docs/debug.md

```diff
@@ -108,3 +108,17 @@ TRAC | Exit data.main.deny = _
 TRAC Redo data.main.deny = _
 TRAC | Redo data.main.deny = _
 ```
+
+## Using trace with other output formats
```

Comment:
Should this documentation be updated or removed since trace works the same for all outputters now?

Reply:
Nevermind, this can be merged as-is. Thanks!